scripts/jest/matchers/toWarnDev.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+function normalizeCodeLocInfo(str) {
+  return str && str.replace(/at .+?:\d+/g, 'at **');
+}
+
+function validator(consoleSpy, expectedWarnings) {
+  if (__DEV__) {
+    if (typeof expectedWarnings === 'string') {
+      expectedWarnings = [expectedWarnings];
+    } else if (!Array.isArray(expectedWarnings)) {
```

Comment:
@bvaughn Are you planning on using Flow inside tests at some point? If so, you could get rid of these checks.

Reply:
True.

And no idea. 😄